### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by dichotommy**
Make the wording a bit nicer.

### CODE REVIEW

File: resources/faq.md [hunk 1]
```diff
@@ -179,11 +179,11 @@ Until our first stable release (v1.0), MeiliSearch minor versions are not compat
 
 ## What are the recommended requirements for hosting a MeiliSearch instance?
 
-**The short answer: we don't know.**
+**The short answer: we don't know yet!**
 
-The complexity involved in the indexing process makes it extremely difficult to accurately estimate the size and memory use of a MeiliSearch database, even knowing the size of the `.JSON` files beforehand.
+Indexing documents is a highly complex process, making it difficult to accurately estimate the size and memory use of a MeiliSearch database. In addition, our engine is improving almost constantly.
```
Comment (guimachiavelli) [inline resources/faq.md:184]: Isn't `almost constantly` a borderline contradiction in terms? If the problem is that some updates actually cause performance regressions (and thus we're not *improving* constantly), perhaps we can do something like:

```suggestion
Indexing documents is a highly complex process, making it difficult to accurately estimate the size and memory use of a MeiliSearch database. At this stage, our engine is changing constantly, which means benchmarks are constantly out of date.
```

Comment (dichotommy) [reply resources/faq.md:184]: Nice change, thanks! 😄

File: resources/faq.md [hunk 2]
```diff
@@ -179,11 +179,11 @@ Until our first stable release (v1.0), MeiliSearch minor versions are not compat
 
 ## What are the recommended requirements for hosting a MeiliSearch instance?
 
-**The short answer: we don't know.**
+**The short answer: we don't know yet!**
 
-The complexity involved in the indexing process makes it extremely difficult to accurately estimate the size and memory use of a MeiliSearch database, even knowing the size of the `.JSON` files beforehand.
+Indexing documents is a highly complex process, making it difficult to accurately estimate the size and memory use of a MeiliSearch database. At this stage, our engine is changing constantly, which means benchmarks are constantly out of date.
```
Comment (guimachiavelli) [inline resources/faq.md:184]: Oops, my bad, repeated constantly.

```suggestion
Indexing documents is a highly complex process, making it difficult to accurately estimate the size and memory use of a MeiliSearch database. At this stage, our engine is changing constantly, which means benchmarks are often out of date.
```

**Review by guimachiavelli — APPROVED** [review]

File: resources/faq.md (hunk 2 above)
Comment (dichotommy) [reply resources/faq.md:184]: I prefer it the way it is 😁

**Pull request merged**